packages/teams-js/src/public/constants.ts

```diff
@@ -165,7 +165,10 @@ export enum ChannelType {
 /** An error object indicating that the requested operation or feature is not supported on the current platform or device.
  * @typedef {Object} SdkError
  */
-export const errorNotSupportedOnPlatform: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
+export const errorNotSupportedOnPlatform: SdkError = {
+  errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM,
+  message: 'Not supported on current platform',
```

Comment:
This new message is causing some E2E tests to fail because they were expecting only an error code in this case. I think we would be fine to add a message here, but updating all of those tests is probably outside the scope of the change you're making. Feel free to go back to error code only (which matches what the rest of our repo does right now) and open a bug for us to add a message here in the future.